cmd/xl-v1-healing-common.go

```
@@ -184,7 +207,11 @@ func xlHealStat(xl xlObjects, partsMetadata []xlMetaV1, errs []error) HealObject
 	// Less than quorum erasure coded blocks of the object have the same create time.
 	// This object can't be healed with the information we have.
 	modTime, count := commonTime(listObjectModtimes(partsMetadata, errs))
-	if count < xl.readQuorum {
+
+	// get Quorum for this object
+	readQuorum, _, _ := objectQuorumFromMeta(xl, partsMetadata, errs)
```

Is it okay to ignore err here?

---

Not sure if you saw this @nitisht ?